db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/BinaryOperatorNode.java

```diff
@@ -977,5 +979,18 @@ public void castLeftOperandAndBindCast(DataTypeDescriptor type) throws StandardE
     public void castRightOperandAndBindCast(DataTypeDescriptor type) throws StandardException {
         castOperandAndBindCast(1, type);
     }
+
+    public void copy(BinaryOperatorNode other) throws StandardException
+    {
+    	super.copy(other);
+        this.operatorType = other.operatorType;
+        this.leftMatchIndexExpr = other.leftMatchIndexExpr;
+        this.rightMatchIndexExpr = other.rightMatchIndexExpr;
+        this.leftMatchIndexExprConglomDesc = other.leftMatchIndexExprConglomDesc;
+        this.rightMatchIndexExprConglomDesc = other.rightMatchIndexExprConglomDesc;
+        this.leftMatchIndexExprColumnPosition = other.leftMatchIndexExprColumnPosition;
+        this.rightMatchIndexExprColumnPosition = other.rightMatchIndexExprColumnPosition;
+        this.xmlQuery = other.xmlQuery;
+    }
```

Comment:
I don't know what's the convention in Java for copying, but a name like `copyFrom()` looks better to me.
Also, indentation is not consistent for `super.copy(other)`, this applies to other `copy()` method as well.

Reply:
Fixed the indentation and changed copy -> copyFrom